apps/prairielearn/src/components/RubricSettings.tsx

```diff
@@ -11,6 +11,24 @@ type RubricItemData = Partial<
   RubricItem & { num_submissions: number; disagreement_count: number | null }
 >;
 
+declare global {
+  interface Window {
+    resetInstructorGradingPanel: () => any;
+    mathjaxTypeset: () => Promise<any>;
```

Comment:
For `mathjaxTypeset` specifically: rather than doing this dance with `window`, you should be able to import `mathjaxTypeset` here from `mathjax.ts`.